Corona-Warn-App/src/main/java/de/rki/coronawarnapp/submission/Symptoms.kt

```diff
@@ -0,0 +1,6 @@
+package de.rki.coronawarnapp.submission
+
+class Symptoms(
+    val startOfSymptoms: StartOfSymptoms,
```

Comment:
What about making `StartOfSymptoms` and `SymptomIndication` subclasses to keep it as one coherent unit (they are not used independently), i.e. `Symptoms.StartOf` and `Symptoms.Indication`. Not a *mustHave*  but seems cleaner to me.